cloudsync/tasks.py

```diff
@@ -1,16 +1,14 @@
 """
 Tasks for cloudsync app
 """
-
 import os
 import re
-from urllib.parse import unquote
-
-import boto3
+import urllib
```

Comment:
yes, but the reason was that urllib is in the standard library and both is installed...

the sequence of imports should be:
```
import <standard library package>
from <standard library package> import <function/class/constants>

import <installed library package>
from < installed library package> import <function/class/constants>

import <local app package>
from < local app package> import <function/class/constants>
```

you can switch the `from` with the imports (other people in the group refer one or the other) as long as it is consistent in the module.